src/map/mobskill.h

```diff
@@ -61,27 +65,28 @@ class CMobSkill
     bool isBloodPactWard() const;
     bool isBloodPactRage() const;
 
-    uint16 getID() const;
-    uint16 getAnimationID() const;
-    uint8  getAoe() const;
-    float  getDistance() const;
-    uint8  getFlag() const;
-    uint16 getAnimationTime() const;
-    uint16 getActivationTime() const;
-    uint16 getMsg() const;
-    uint16 getAoEMsg() const;
-    uint16 getValidTargets() const;
-    int16  getTP() const;
-    uint8  getHPP() const;
-    uint16 getTotalTargets() const;
-    uint32 getPrimaryTargetID() const;
-    uint16 getMsgForAction() const;
-    float  getRadius() const;
-    int16  getParam() const;
-    uint8  getKnockback() const;
-    uint8  getPrimarySkillchain() const;
-    uint8  getSecondarySkillchain() const;
-    uint8  getTertiarySkillchain() const;
+    uint16                             getID() const;
+    uint16                             getAnimationID() const;
+    uint8                              getAoe() const;
+    float                              getDistance() const;
+    uint8                              getFlag() const;
+    uint16                             getAnimationTime() const;
+    uint16                             getActivationTime() const;
+    uint16                             getMsg() const;
+    uint16                             getAoEMsg() const;
+    uint16                             getValidTargets() const;
+    int16                              getTP() const;
+    uint8                              getHPP() const;
+    const std::vector<CBattleEntity*>& getTargets() const;
```

Comment:
You can make this auto and trailing return (->) to avoid changing all these lines with clang-format